spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplateTests.java

```diff
@@ -192,6 +196,15 @@ public void deleteByIdTest() {
 		verify(this.datastore, times(1)).delete(same(key1));
 	}
 
+	@Test
+	public void deleteAllByIdTest() {
+		Key key1 = createFakeKey();
+		when(this.objectToKeyFactory.getKeyFromId(same(key1), any())).thenReturn(key1);
+		this.datastoreTemplate.deleteAllById(ImmutableList.of(key1, key1),
```

Comment:
would be better to have 2 distinct keys

Reply:
done

Reply:
sorry about that! (i didn't know it emailed you even if you weren't previously on the specific PR. )

Reply:
Well, I'm a commiter on the project, so that is my responsibility to monitor (watch) all the PRs and issues in the project.

I'm not commenting on your PRs because I might just don't understand stuff and my questions for explanation might be just a noise, but that's all doesn't mean I don't review PRs 😄 

Anyway I believe that `done` is a spam not only for me. GitHub has been improved significantly over years to make our, reviewers, life easier.
That's why I prefer to have silent agreement and a proper next commit's message to determine all your changes in a single email.

Thanks and sorry for this spam from me 😉